storm-server/src/main/java/org/apache/storm/scheduler/resource/ResourceAwareScheduler.java

```diff
@@ -248,6 +255,11 @@ private void scheduleTopology(TopologyDetails td, Cluster cluster, final User to
                     + topologySchedulingResources.getRemainingRequiredResourcesMessage());
     }
 
+    public Map<String, Set<String>> getEvictedTopologiesMap() {
```

Comment:
I would add a comment like `This is not stable. It's subject to change` since we don't really use it elsewhere besides unit test currently. It might change depending on how we want to use the evictedTopologies information. As @bipinprasad  said, we might want to put it into the scheduleResult in the future.

Reply:
Added.

Reply:
Good comment